Code/GraphMol/MarvinParse/MarvinWriter.cpp

```diff
@@ -429,8 +429,13 @@ class MarvinCMLWriter {
         marvinAtom->formalCharge = atom->getFormalCharge();
 
         unsigned int nRadEs = atom->getNumRadicalElectrons();
+        // value of radical electrons has to be one of the expected values or it
+        // is ignored
         if (nRadEs != 0) {
-          marvinAtom->radical = radicalElectronsToMarvinRadical.at(nRadEs);
+          if (radicalElectronsToMarvinRadical.find(nRadEs) !=
+              radicalElectronsToMarvinRadical.end()) {
+            marvinAtom->radical = radicalElectronsToMarvinRadical.at(nRadEs);
+          }
```

Comment:
This can be more efficient if you only search through the map once:
```suggestion
          if (const auto iter = radicalElectronsToMarvinRadical.find(nRadEs);
              iter != radicalElectronsToMarvinRadical.end()) {
            marvinAtom->radical = iter->second;
          }
```